geologist/child-theme.json

```diff
@@ -357,28 +357,28 @@
 			"h3": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "min(max(28px, 5vw), 38px)",
+					"fontSize": "var(--wp--preset--font-size--extra-large)",
 					"lineHeight": 1.2
 				}
 			},
 			"h4": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "20px",
+					"fontSize": "var(--wp--preset--font-size--large)",
 					"lineHeight": 1.4
 				}
 			},
 			"h5": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "18px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
 					"lineHeight": 1.4
 				}
 			},
 			"h6": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "16px",
+					"fontSize": "var(--wp--preset--font-size--small)",
```

Comment:
Should this one be tiny?

Reply:
Same as the above. Quadrat - https://github.com/Automattic/themes/pull/4683/files#diff-a1fd00442cdce71e5586a4cc5607a542027fe9cbc72ab9aa6f182d5b8f524142R406